v2/src/components/recipeBoxes/guides.json

```diff
@@ -0,0 +1,79 @@
+{
+  "auth_modes": [
+    {
+      "title": "Email password Login",
+      "icon": "emailPass",
```

Comment:
The spacing to the right of the select is wrong. Also the designs have a minimum width for the select it seems

<img width="169" alt="Screenshot 2023-03-13 at 6 43 05 PM" src="https://user-images.githubusercontent.com/18233774/224712144-171eec2a-dc55-4a60-9d85-68644e86b1af.png">